express/src/routes/account.ts

```diff
@@ -74,5 +74,5 @@ router
         });
     })
     .post((req, res) => {
-        res.redirect("change-phone-number/enter-text-code");
+        res.redirect("/account/change-phone-number/enter-text-code");
```

Comment:
```suggestion
        res.redirect("/account/change-phone-number/enter-text-code");
```
This is not needed - this is a relative path. It will automatically redirect to `/account/....` if the path has not leading `/`